common/reducers/wallet.ts

```diff
@@ -1,17 +1,13 @@
 import { SetBalanceFullfilledAction } from 'actions/wallet/actionTypes';
-import {
-  SetTokenBalancesAction,
-  SetWalletAction,
-  WalletAction,
-  TypeKeys
-} from 'actions/wallet';
+import { SetTokenBalancesAction, SetWalletAction, WalletAction, TypeKeys } from 'actions/wallet';
 import { TokenValue } from 'libs/units';
 import { BroadcastTransactionStatus } from 'libs/transaction';
-import { IWallet, Balance } from 'libs/wallet';
+import { IWallet, Balance, WalletConfig } from 'libs/wallet';
 import { getTxFromBroadcastTransactionStatus } from 'selectors/wallet';
 
 export interface State {
   inst?: IWallet | null;
+  config?: WalletConfig | null;
```

Comment:
Confused about the shape of WalletConfig: `tokens?: string[] | null;` 

`tokens` is already a property - will that be removed in favor of WalletConfig? 

I would have expected something more along the lines of:
```
interface Accounts {
    [key: string]: {
        [token: string]: TokenObj
    }
}

accounts?: Accounts | null```

Reply:
Yeah, now that I look at it, the nomenclature is pretty bad. What is currently on `state.wallet.tokens` is actually just token _balances_ (As well as an error state.) I should probably rename that property `tokenBalances` and just make them balance objects.

The tokens object on the config would just be an array of token symbols, i.e. `["BAT", "GNT", ...]` that will flag which tokens we're currently tracking for that wallet.

I didn't want to store full token data in local storage (address, decimal etc.) in case we ever change it or expand on it (icons, written names, website addresses etc.) so that it just looks up the token object each time.

Reply:
And to be more clear about why this is under `WalletConfig`, I think the future of `WalletConfig` will look something like:

```
interface WalletConfig {
  name?: string | null;
  tokens?: string[] | null;
  type?: keyof WalletTypes | null;
  syncConfigToCloud?: boolean;
}
```

that keeps track of user preferences or other UX-centric things to make wallets more persistent.

Reply:
Got it, much more clear now.